src/components/Layout/Stepper/index.jsx

```diff
@@ -0,0 +1,167 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { withRouter } from 'react-router-dom';
+import { observer } from 'mobx-react';
+import { translate } from 'react-i18next';
+import _ from 'lodash';
+import Loading from 'components/Loading';
+
+import { DocLink, Icon } from 'components/Base';
+
+import styles from './index.scss';
+
+@translate()
+@observer
+class LayoutStepper extends Component {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+    name: PropTypes.string,
+    store: PropTypes.shape({
```

Comment:
The name is not good, rename it as `stepOption`

Reply:
ok